src/Sulu/Bundle/AdminBundle/Resources/js/components/Block/block.scss

```diff
@@ -72,7 +72,7 @@ $blockIconsColor: $gray;
 }
 
 .types {
-    width: 120px;
+    width: 150px;
```

Comment:
i think we can just use `min-width: 120px;` here. that would fix the problem for even longer type names